### PR TITLE
Gm qtype add units

### PR DESCRIPTION
**Pull request by davidhughes1-ccs**
As requested by Tom to allow display of unit type for numeric questions and answers

### CODE REVIEW

**Comment by tommyb82** [comment]
Thanks @davidhughes1-ccs - I'll check with the UI guys that this provides them with sufficient information - should do I think 👍

File: gm-decision-tree/decision-tree-service.yaml [hunk 1]
```diff
@@ -239,6 +241,14 @@ components:
           type: string
           enum: [number, text, date, daterange]
           description: The type of conditional input
+        unit: 
+          $ref: '#/components/schemas/Unit'
+
+    Unit:        
+      type: string
+      enum: [currency|quantity|days|weeks|months|years]
```
Comment (tommyb82) [inline gm-decision-tree/decision-tree-service.yaml:249]: Should be comma-separated strings

File: guided-match/guided-match-service.yaml [hunk 2]
```diff
@@ -377,6 +377,13 @@ components:
           type: string
           enum: [boolean,list,multiselect,number,text,date,daterange]
           description: "The type of the question.  Described in detail in the HLD section 7.1.2.2"
+        unit: 
+          $ref: '#/components/schemas/Unit'
+
+    Unit:        
+      type: string
+      enum: [currency|quantity|days|weeks|months|years]
```
Comment (tommyb82) [inline guided-match/guided-match-service.yaml:385]: Should be comma-separated strings 😉

**Comment by davidhughes1-ccs** [comment]
Have made the enum changes requested - thanks

**Pull request merged**